packages/parser-javascript/src/parser.ts

```diff
@@ -30,7 +30,7 @@ export default class JavascriptParser extends WebhintParser<ScriptEvents> {
         try {
             await this.engine.emitAsync(`parse::start::javascript`, { resource });
 
-            const options: Options = { locations: true, ranges: true };
+            const options: Options = { ecmaVersion: 2020, locations: true, ranges: true };
```

Comment:
Should we consider using "latest" here per https://github.com/acornjs/acorn/tree/master/acorn#interface? Otherwise we're going to need some way to remember to bump this to the next version later.

```suggestion
            const options: Options = { ecmaVersion: "latest", locations: true, ranges: true };
```

Reply:
Sure, I use `2020`  because that was what the warning was saying hehehe.